charts/datasaur/templates/start/document-converter-deployment.yaml

```diff
@@ -32,9 +32,9 @@ spec:
 {{ include "imagePullSecretTemplate" . | nindent 6 }}
       containers:
 {{- if .Values.dockerHub.enabled }}
-        - image: thecodingmachine/gotenberg:7.10.2
+        - image: thecodingmachine/gotenberg:8.2.2
 {{- else }}
-        - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}thecodingmachine/gotenberg:7.10.2
+        - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}thecodingmachine/gotenberg:8.2.2
```

Comment:
tolong apply changes dr PR mike juga ya @andika-william 

https://github.com/datasaur-ai/helm-charts/pull/29/files

Reply:
keep up master dulu yak @andika-william

Reply:
okede, on my way @tonoman3g

Reply:
rebased to current master (main)

aku liat PR audrey yg merged juga sudah keeup sama punya michael. 
Jadi harusnya master/main sudah keeup juga ya ka ton